<ui version="4.0" >
 <class>DirStatForm</class>
 <widget class="QWidget" name="DirStatForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QTextEdit" name="textEdit" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="sizeIncrement" >
          <size>
           <width>1</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="acceptDrops" >
          <bool>false</bool>
         </property>
         <property name="toolTip" >
          <string comment="Joblog Window" />
         </property>
         <property name="statusTip" >
          <string comment="Joblog Window" />
         </property>
         <property name="whatsThis" >
          <string comment="Joblog Window" />
         </property>
         <property name="horizontalScrollBarPolicy" >
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="autoFormatting" >
          <set>QTextEdit::AutoNone</set>
         </property>
         <property name="tabChangesFocus" >
          <bool>false</bool>
         </property>
         <property name="documentTitle" >
          <string comment="Joblog Window" />
         </property>
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>221</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>221</width>
           <height>100</height>
          </size>
         </property>
         <property name="title" >
          <string>Refresh Timer</string>
         </property>
         <widget class="QSpinBox" name="spinBox" >
          <property name="geometry" >
           <rect>
            <x>20</x>
            <y>60</y>
            <width>111</width>
            <height>24</height>
           </rect>
          </property>
          <property name="minimum" >
           <number>5</number>
          </property>
          <property name="maximum" >
           <number>999</number>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox" >
          <property name="geometry" >
           <rect>
            <x>20</x>
            <y>20</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text" >
           <string>Do Refresh</string>
          </property>
         </widget>
         <widget class="QLCDNumber" name="timerDisplay" >
          <property name="geometry" >
           <rect>
            <x>110</x>
            <y>20</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="dirstatlayoutWidget" >
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="scheduledLabel" >
         <property name="layoutDirection" >
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:13pt; font-weight:600;">Scheduled Jobs&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="scheduledTable" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2" >
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="runningLabel" >
         <property name="layoutDirection" >
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:13pt; font-weight:600;">Running Jobs&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="runningTable" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="terminatedLabel" >
         <property name="layoutDirection" >
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:13pt; font-weight:600;">Terminated Jobs&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="terminatedTable" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionRefresh" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/view-refresh.png</iconset>
   </property>
   <property name="text" >
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionCancelRunning" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/utilities-terminal.png</iconset>
   </property>
   <property name="text" >
    <string>Cancel Selected Running Jobs</string>
   </property>
  </action>
  <action name="actionDisableScheduledJob" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/utilities-terminal.png</iconset>
   </property>
   <property name="text" >
    <string>Disable Scheduled Job</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../main.qrc" />
 </resources>
 <connections/>
</ui>
